content/pytorch/concepts/tensors/terms/zeros/zeros.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'ZEROS'
+Description: 'Initializing a new tensor with zeros.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+  - 'AI'
+Tags:
+  - 'AI'
+  - 'Data Structures'
+  - 'Arrays'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+  - 'paths/machine-learning'
+---
```

Comment:
```suggestion
---
Title: '.zeros()'
Description: 'Initializes a new tensor filled with zeros, with a specified shape.'
Subjects:
  - 'AI'
  - 'Data Science'
  - 'Machine Learning'
Tags:
  - 'AI'
  - 'Arrays'
  - 'Data Structures'
  - 'Deep Learning'
CatalogContent:
  - 'intro-to-py-torch-and-neural-networks'
  - 'py-torch-for-classification'
  - 'paths/data-science'
---
```

Reply:
Modifications:
1. title is `.zeros`
2. description should start with verbs
3. Tags and subjects should be in alphabetical order